wagtail/wagtailadmin/templates/wagtailadmin/shared/collection_chooser.html

```diff
@@ -0,0 +1,18 @@
+{% load i18n %}
+
+<li>
+    <div class="field choice_field select">
+        <label for="collection_chooser_collection_id">{% trans "Collection" %}:</label>
```

Comment:
Sorry, problem not in this template, but you can reproduce this on image editing page when `$(window).width() == 1350`.